recipe/meta.yaml

```diff
@@ -57,8 +59,14 @@ test:
 about:
   home: https://pypi.org/project/pdm/
```

Comment:
I would use https://pdm.fming.dev/ for both home and doc_url and https://github.com/pdm-project/pdm for dev_url.